src/main/java/mate/jdbc/model/Car.java

```diff
@@ -0,0 +1,68 @@
+package mate.jdbc.model;
+
+import java.util.List;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers;
+
+    public Car() {
+    }
+
+    public Car(String model, Manufacturer manufacturer, List<Driver> drivers) {
+        this.model = model;
+        this.manufacturer = manufacturer;
+        this.drivers = drivers;
+    }
+
+    public Car(Long id, String model, Manufacturer manufacturer, List<Driver> drivers) {
+        this.id = id;
+        this.model = model;
+        this.manufacturer = manufacturer;
+        this.drivers = drivers;
+    }
```

Comment:
build your logic either on constructors or on setters, 3 overloaded is too much, try to remove at least one of them